docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -1,7 +1,240 @@
+[holocron-readme]: https://github.com/americanexpress/holocron/tree/master/packages/holocron/README.md
+[holocron-module-route-readme]: https://github.com/americanexpress/holocron/tree/master/packages/holocron-module-route/README.md
+[one-app-bundler-readme]: https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler/README.md
+[one-app-router-readme]: https://github.com/americanexpress/one-app-router/README.md
+[frank-lloyd-root]: ../../prod-sample/sample-modules/frank-lloyd-root/0.0.0/README.md
+[franks-burgers]: ../../prod-sample/sample-modules/franks-burgers/0.0.0/README.md
+
 [👈 Return to Overview](./Recipes.md)
 
 # Code Splitting Using Holocron
 
-Recipe is forthcoming.
+For One App, Holocron modules are a formal way of code-splitting for a modern,
+micro front-end in the One App ecosystem. Utilizing the module map as a system
+which promotes synergy and asynchrony across teams and collaborators on
+a single application.
+
+With Holocron each individual module is by definition, a code split bundled
+component with language packs that can be combined and composed based on our
+needs. Each module is versioned, configurable, routable and can be further
+split into smaller chunks using dynamic importing.
+
+Like `@loadable/component` and `react-loadable`, `holocron` is a complete code-splitting
+solution that works in both client and server side, however there is no need to scaffold
+a build infrastructure to support code-splitting when we use `one-app-bundler`.
+
+**[Read the `holocron` documentation.][holocron-readme]**
+
+**[Read the `one-app-bundler` documentation.][one-app-bundler-readme]**
```

Comment:
Let's use our More Information pattern for referencing outside information:

Example:
```md
**📘 More Information**
* Library: [One App Bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler)
* CLI Command: [npm start](./CLI-Commands.md#start-usage)
* [Environment Variables](./Environment-Variables.md)
* [Development Tools](./Development-Tools.md)
```

Reply:
Thank you for sharing the example! I'll update to match the pattern.

Reply:
updated in cb5e958